hrui/hrui.php

```diff
@@ -934,3 +944,73 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+/**
+ * Creates Help Menu in navigation bar
+ *
+ * @param Array $menu List of available menu items
+ */
+function _hrui_createHelpMenu(&$menu) {
+  _hrui_civix_insert_navigation_menu($menu, '', array(
```

Comment:
Looks a lot cleaner with only the required parameters :smiley_cat: 

One final change, instead of `array()` could you use the short syntax, `[]`